### DISCUSSION TITLE
Miss mkdir tests/python/unittest/data before executing tests in test_image.py module

### DISCUSSION THREAD

**Discussion by s0nghuiming** [apache/incubator-mxnet#16946] [Q&A] [answered]
## Description
Miss mkdir tests/python/unittest/data before executing tests in test_image.py module
This will lead test_image failed two cases, they are:
  - test_image#TestImage#test_imageiter
  - test_image#TestImage#test_image_detiter

## To Reproduce
(If you developed your own code, please provide a short script that reproduces the error. For existing examples, please provide link.)

### Steps to reproduce
1. cd tests/python/unittests && python test_image.py
2. check result of the output. two cases fail.
**test_imageiter** and **test_image_detiter**

## What have you tried to solve it?
1. Manual mkdir tests/python/unittests/data
2. retry the command.

**Comment by samskalicky** [comment]
@mxnet-label-bot update [Question, HowTo, Test]

**Comment by szha** [comment, marked as answer]
The test infrastructure is now moved to pytest with the tmpdir fixture, so this should no longer be an issue.
